uber/templates/regform.html

```diff
@@ -89,7 +91,7 @@
 
 {% if c.PAGE == 'confirm' %}
     <div class="form-group">
-        <label for="email" class="col-sm-2 control-label">Legal Name</label>
+        <label for="full_name" class="col-sm-2 control-label">Legal Name</label>
```

Comment:
It's also useful even for non-accessibility reasons.  Like if you click on the label with the appropriate ``for`` label, it gives the input field focus.  That's helpful if someone wants to click on a field but isn't super-precise with the mouse.